httpdocs/ajax/animation/create.php

```diff
@@ -63,12 +63,14 @@ function parse_mf_string($line, $parameter) {
       $title = parse_sf_string($line, 'title');
       $info = parse_mf_string($line, 'info');
       $description = implode("\n", $info);
-    }
+    } else if (substr($line, 0, 30) == '<meta name="version" content="')
```

Comment:
The x3d files I am testing with have: `<meta name="generator" content="Webots" />`
Will this be a problem? I have not come across this meta header node yet.